configs/rec/svtr/svtr_tiny_ch.yaml

```diff
@@ -0,0 +1,180 @@
+system:
+  mode: 0 # 0 for graph mode, 1 for pynative mode in MindSpore
+  distribute: True
+  amp_level: O2
+  amp_level_infer: O2 # running inference in O2 mode
+  seed: 42
+  log_interval: 100
+  val_while_train: True
+  drop_overflow_update: False
+  ckpt_max_keep: 5
+
+common:
+  character_dict_path: &character_dict_path mindocr/utils/dict/ch_dict.txt
+  num_classes: &num_classes 6624 # num_chars_in_dict + 1
+  max_text_len: &max_text_len 40
+  use_space_char: &use_space_char False
+  batch_size: &batch_size 256
+
+model:
+  type: rec
+  transform: null
+  backbone:
+    name: SVTRNet
+    pretrained: False
+    img_size: [32, 320]
+    out_channels: 96
+    patch_merging: Conv
+    embed_dim: [64, 128, 256]
+    depth: [3, 6, 3]
+    num_heads: [2, 4, 8]
+    mixer:
+      [
+        "Local",
+        "Local",
+        "Local",
+        "Local",
+        "Local",
+        "Local",
+        "Global",
+        "Global",
+        "Global",
+        "Global",
+        "Global",
+        "Global",
+      ]
+    local_mixer: [[7, 11], [7, 11], [7, 11]]
+    last_stage: True
+    prenorm: False
+  neck:
+    name: Img2Seq
+  head:
+    name: CTCHead
+    out_channels: *num_classes
+
+postprocess:
+  name: RecCTCLabelDecode
+  character_dict_path: *character_dict_path
+  use_space_char: *use_space_char
+
+metric:
+  name: RecMetric
+  main_indicator: acc
+  lower: False
+  character_dict_path: *character_dict_path
+  ignore_space: True
+  print_flag: False
+
+loss:
+  name: CTCLoss
+  pred_seq_len: 80 # 320 / 4
+  max_label_len: *max_text_len # this value should be smaller than pre_seq_len
+  batch_size: *batch_size
+
+scheduler:
+  scheduler: warmup_cosine_decay
+  min_lr: 0.00001
+  lr: 0.001
+  num_epochs: 60
+  warmup_epochs: 6
+  decay_epochs: 54
+
+optimizer:
+  opt: adamw
+  grouping_strategy: svtr
+  filter_bias_and_bn: False
+  weight_decay: 0.05
+
+loss_scaler:
+  type: dynamic
+  loss_scale: 512
+  scale_factor: 2.0
+  scale_window: 1000
+
+train:
+  ckpt_save_dir: ./tmp_rec
+  dataset_sink_mode: False
+  clip_grad: True
+  clip_norm: 0.1
+  dataset:
+    type: LMDBDataset
+    dataset_root: path/to/data_lmdb_release/
+    data_dir: training/
+    label_file: null
+    sample_ratio: 1.0
+    shuffle: True
+    filter_max_len: True
+    max_text_len: *max_text_len
+    transform_pipeline:
+      - DecodeImage:
+          img_mode: BGR
```

Comment:
Consider setting bgr_to_rgb: False, and bgr_to_rgb: False, which can has some influence on computing results but save some time.